axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceLineServiceImpl.java

```diff
@@ -304,7 +304,12 @@ public Unit getUnit(Product product, boolean isPurchase) {
     productInformation.put("totalPack", null);
     productInformation.put("packPriceSelect", 0);
     productInformation.put("typeSelect", InvoiceLineRepository.TYPE_NORMAL);
-    if (appAccountService.getAppInvoice().getIsEnabledProductDescriptionCopy()) {
+    if ((invoice.getOperationTypeSelect() == 3
```

Comment:
Please, don't use hard coded value. 

Better to call the InvoiceToolService.isPurchase(Invoice invoice) method to know if it's a supplier or customer invoice.
https://github.com/axelor/axelor-business-suite/blob/master/axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceToolService.java#L71